dashboard/test/models/regional_partner_test.rb

```diff
@@ -190,13 +190,13 @@ class RegionalPartnerTest < ActiveSupport::TestCase
     create :regional_partner_program_manager, regional_partner: regional_partner, program_manager: partner_organizer
 
     future_partner_workshops = [
-      create(:workshop, organizer: partner_organizer, sessions_from: Date.today),
+      create(:workshop, organizer: partner_organizer, sessions_from: Time.zone.today),
       create(:workshop, organizer: partner_organizer, sessions_from: Date.tomorrow)
```

Comment:
Why was "Date" in the first line changed but not the second? If there's no `Time.zone.tomorrow` function could we do `Time.zone.today + 1.day`?

Reply:
Oh, good question, thanks for asking; I forgot to update the PR description for this. The default `EnforcedStyle` option for this is `flexible`, which means only instances of `Date.today` are prohibited even though `Time.zone.tomorrow` and others do indeed exist. Originally I did indeed replace all instances of `current`, `yesterday`, and `tomorrow` as well, but ran into some issue and decided to scope this change to just what the rule requires by default.

I think I'd be in favor of us setting `EnforcedStyle` to  `strict` and updating everything else, but decided to save that for follow-up work